modules/javafx.controls/src/main/java/javafx/scene/control/skin/SplitPaneSkin.java

```diff
@@ -65,6 +64,12 @@
     private ObservableList<Content> contentRegions;
     private ObservableList<ContentDivider> contentDividers;
     private boolean horizontal;
+    /**
+     * Flag which is used to determine whether we need to request layout when a divider position changed or not.
+     * E.g. We don't want to request layout when we are changing the divider position in
+     * {@link #layoutChildren(double, double, double, double)} since we are currently doing the layout.
+     */
+    private boolean duringLayout;
```

Comment:
would like a reference to the bug this fixes

Reply:
Added it